README.md

```diff
@@ -166,6 +166,31 @@ def lambda_handler(event, context):
     }
 ```
 
+## Replacing the SSM client
+
+You can replace the SSM client for testing by setting the `SSMParameterStore.SESSION_FACTORY` and / or the `SSMParameterStore.CLIENT_FACTORY` class variables.
+
+For example, if you are using the `placebo` library to mock out `boto3` behaviour:
+
+```python
+import boto3
+import placebo
+
+from ssm_cache import SSMParameterStore
+
+def session_factory():
+    session = boto3.Session()
+
+    pill = placebo.attach(session, data_path='/path/to/response_directory')
+    pill.playback()
+
+    return session
+
+SSMParameterStore.SESSION_FACTORY = staticmethod(session_factory)
```

Comment:
This is actually ugly and not very user-friendly. Maybe we can make it easier to use with an injection method?

Something like this:

```python
SSMParameterStore.inject_session_factory(session_factory)
```

`SSMParameterStore` will manage the given callable internally, so we don't need to expose class variable to the final user. This will change a bit @benkehoe's implementation, but I think it's worth it.

It makes sense given that factories will be stateless anyway. What do you think?

Reply:
👍